src/it/scala/com/ing/wbaa/testkit/docker/DockerCephS3Service.scala

```diff
@@ -31,6 +37,16 @@ trait DockerCephS3Service extends DockerKit {
     )
     .withCommand("demo")
 
+  // Settings to connect to S3 storage, we have to wait for the docker container to retrieve the exposed port
+  lazy val gargoyleStorageS3SettingsFuture: Future[GargoyleStorageS3Settings] =
+    cephContainer.getPorts()(docker = dockerExecutor, ec = dockerExecutionContext)
+      .map { portMapping =>
+        new GargoyleStorageS3Settings(testSystem.settings.config) {
+          override val storageS3Authority: Uri.Authority =
+            Uri.Authority(Uri.Host("127.0.0.1"), portMapping(cephInternalPort))
```

Comment:
windows IP problem here :) 
so maybe create a file it/resources/application.conf and add 
${?GARGOYLE_IT_DOCKER_IP)

Reply:
Am working on a version that starts up using docker-compose, which would remove these docker services here. (won't be started by code but using docker-compose)